### ISSUE TITLE
Access the editing layout and numeric keyboard more easily...!

### ISSUE THREAD

**Issue by TheThunderspy** [florisboard/florisboard#779]
It would be way more cooler a shortcut configurable in keyboard settings.
Like double pressing the space bar or hold the Enter key to summon both Numeric keyboard and editting layout...!
Or including it on the Smart Bar.

**Comment by victorbnl** [comment]
That's not a bad idea! Do you have examples of this feature existing already on other keyboards?

**Comment by pokegamer5547** [comment]
Not sure if this is the place to ask but when will editing layout will be implemented again? It says "see #216" but i can't find that issue anywhere.

**Comment by Soulthym** [comment]
@victorbnl @TheThunderspy  
I am not sure that is the same idea, but I was thinking of a vim-like macro/register system whers you can record some movements you do often and trigger them with a dedicated macro key, followed by a letter.


For this in vim they use:  
- letters for registers(reg)
```vim
<esc><q><reg> <...doing + recording...><q>
> to record into register <reg>

<esc><q><reg><q>
> to delete (same as recording an empty macro)

<esc><@><reg>
> to replay the macro from <reg>
```
-------
We could have something similar to record and manage macros, and a special macro bar/keyboard that's customizable?  
I would personally love to use a board with macros that's fully open-source.

**Comment by victorbnl** [comment]
Yep of course, this is a great idea but this has nothing to do with accessing the editing layout a numeric keyboard more easily, which is the point of this issue.

This should go to a new issue instead

**Comment by victorbnl** [comment]
I created the issue at #946
